HW2Question1.cpp

```diff
@@ -0,0 +1,77 @@
+//
+//  HW2Question1.cpp
+//  Codu
+//
+//  Created by Farida Eid on 5/27/17.
+//  Copyright © 2017 Farida Eid. All rights reserved.
+//
+
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+struct treeNode
+{
+    int value;
+    treeNode *left;
+    treeNode *right;
+    
+    treeNode(int v):value(v), left(NULL), right(NULL){}
+
+};
+
+bool printAncestors(treeNode *head, int key, vector<int>& answer)
```

Comment:
Please add a comment to the function, explaining what it does, what is expected from the tree, how it behaves in special cases.